db/migrate/20230908185452_add_state_to_accession.rb

```diff
@@ -0,0 +1,7 @@
+class AddStateToAccession < ActiveRecord::Migration[7.0]
+  def change
+    add_column :accessions, :state, :string, null: false, default: 'in_progress'
```

Comment:
is the reason for the default that an `accessions` row only gets created once the accessioning is launched for the druid?

Reply:
Correct.